src/main/java/com/it4045/pokemonviewer/enterprise/dao/PokemonDAO.java

```diff
@@ -23,4 +23,17 @@
         return pokemonList;
     }
 
+    public Pokemon fetch(int id) {
+
+        return new Pokemon();
+    }
+
+    public Pokemon save(Pokemon pokemon) {
+        return new Pokemon();
+    }
+
+    public void delete(int id) {
+
+    }
+
```

Comment:
goes along with IPokemonDAO additions. Created stubs for you to fill in remaining logic for these CRUD operations.

Reply:
This looks like the start of a new feature.  Avoid adding new features in a code review, as a good code review should merge to master, and only features that meet the team's Definition of Done should be merged to master.  If functionality is introduced and not completed, it adds to technical debt, as it's unused code.